src/System.Management.Automation/resources/Modules.resx

```diff
@@ -165,6 +165,9 @@
   <data name="InvalidModuleSpecificationMember" xml:space="preserve">
     <value>The hashtable describing a module contains one or more members that are not valid.  The valid members are ({0}). Remove the members that are not valid ({1}), then try again.</value>
   </data>
+  <data name="ModuleSpecificationMemberIsLessThanOther" xml:space="preserve">
```

Comment:
The error message should ideally be slightly more specific. Something like:
> The hashtable describing a module cannot have its ModuleVersion ({0}) greater than its MaximumVersion ({1}).

Reply:
Done.